### DISCUSSION TITLE
Using it with newer whisper models like whisper-large-v3?

### DISCUSSION THREAD

**Discussion by empz** [linto-ai/whisper-timestamped#209] [Q&A] [answered]
Is this library compatible with newer Whisper models (v2, v3)?

I've tried using the [whisper-large-v3](https://huggingface.co/openai/whisper-large-v3), but `transcribe` throws the following error:

```
RuntimeError: Given groups=1, weight of size [1280, 128, 3], expected input[1, 80, 3000] to have 128 channels, but got 80 channels instead
```

**Comment by Jeronymous** [comment, marked as answer]
It should work with large V3
What is your version of whisper-timespamped? And openai-whisper.
 If it's not the latest version for both, updating should fix.

If if fails with the latest version, please provide a minimal code to reproduce.

**Reply by empz** [reply]
Uhm, it's working now. No idea why though. I've just tried this for the first time yesterday in 2 different platforms (Google Colab and Modal) and it was failing. I guess an older version of some package was being installed for some reason...
